rfc/src/rfcs/0013-list.md

```diff
@@ -141,9 +150,90 @@ If there is no result for a given field (e.g., there are no contracts), Kani wil
 
 ## Software Design
 
-We will add a new subcommand to `kani-driver`.
+### Metdata Changes
+We introduce a new `ContractedFunction` struct to `kani_metadata`:
+```rust
+pub struct ContractedFunction {
```

Comment:
TBH, I don't think you need to add this level of detail. I think saying high-level that the metadata will now include a list of functions that contains contract and information about their harnesses.